### DISCUSSION TITLE
A couple marketing site built with UniformCSS

### DISCUSSION THREAD

**Discussion by stillingdesign** [ThinkUniform/uniformcss#38] [Show and tell]
Hey Jin and the UniformCSS community, thought I'd share a couple of sites that I've recently worked on that use Uniform:

[Rewatch.com](https://rewatch.com/)
[FireHydrant.com](https://firehydrant.com/)

Uniform makes building this stuff sooooo much faster/easier/better, thank you ❤️ .

**Comment by conormuirhead** [comment]
Congrats on launching the new FireHydrant website @stillingdesign!

**Reply by stillingdesign** [reply]
Thanks @conormuirhead! The three week timeline was a little rough - very thankful for Uniform  🙌  and some re-used/purposed config code from you, Conor 😄 .

**Comment by jinsupark** [comment]
This is absolutely awesome to see, the design of these sites look really great!

Thanks for sharing this! ❤️ Truly made my day 😀 

I'm about the launch a new website for uniformcss.com would it be alright if I add the Firehydrant logo on this new site? @stillingdesign

**Reply by stillingdesign** [reply]
Hey @jinsupark I'm glad you like them! And thank you again, Uniform makes building this stuff so much faster and easier 🙌 .  

I just double checked with my manager (VP of Marketing) and they said it's cool to use the FireHydrant logo on the Uniform Site!

I'm currently working on updating the homepage of FireHydrant ([sneak peak](https://deploy-preview-32--firehydrant-dotcom.netlify.app/)) - still have to add decoration and graphics. Also working on a new [personal/portfolio site](https://mikes2023.netlify.app/) - balancing self-promotion and reality is awkward. All may or may not be heavily inspired by Linear 😅.

**Reply by stillingdesign** [reply]
Just shipped that homepage! Also, I've been working on a side-project redesigning [the docs](https://deploy-preview-10--zingy-lamington-7ce1aa.netlify.app/) with a co-worker. Also using Uniform 😄

**Reply by jinsupark** [reply]
WOW @stillingdesign this is absolutely beautifully executed, congratulations and nice work!

Absolutely awesome to see this providing value in the wild, thank you very much for sharing :)

Thanks for getting that approval! UniformCSS V2 website and UniformIcons is launching very soon, stay tuned!
